.github/workflows/validate_json.yml

```diff
@@ -28,26 +28,27 @@ jobs:
           pip install pyyaml
           cd schema
           make
-          echo {} > test.json
       - name: Run import tools on test data
         run: |
           # TODO: use requirements file
           pip install pyyaml pandas humanfriendly tqdm exifread pillow
-
           cd conversion_tools/cwfid_to_json/
           curl -L https://github.com/cwfid/dataset/archive/36290d0.tar.gz | tar xzv
           mv dataset-36290d0*/* .
-          python cwfid_to_json.py
-
+          mv images cwfid_images
+          python cwfid_to_json.py --image-dir cwfid_images
           cd ../deepweeds_to_json
           curl -L https://github.com/AlexOlsen/DeepWeeds/archive/51e3fab.tar.gz | tar xzv
           mv DeepWeeds-51e3fab*/* .
           # fake some images: make them all be a pic from cwfid
           mkdir deepweeds_images_full
-          for fname in $(cut -d, -f1 < labels.csv); do ln -s ../../cwfid_to_json/images/001_image.png deepweeds_images_full/$fname; done
+          for fname in $(cut -d, -f1 < labels.csv); do ln -s ../../cwfid_to_json/cwfid_images/001_image.png deepweeds_images_full/$fname; done
           python deepweeds_to_json.py --labels-dir labels --image-dir deepweeds_images_full -o deepweeds_imageinfo.json
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10'
+      - run: npm install -g ajv-cli
```

Comment:
this replaces the (limited) functionality of the previous orrosenblatt validator?